store/datastore/utils.go

```diff
@@ -38,34 +39,33 @@ func rebind(query string) string {
 
 // helper function that converts a simple repsitory list
 // to a sql IN statment.
-func toList(listof []*model.RepoLite) (string, []interface{}) {
-	var size = len(listof)
-	if size > 999 {
-		size = 999
-		listof = listof[:999]
-	}
-	var qs = make([]string, size, size)
-	var in = make([]interface{}, size, size)
-	for i, repo := range listof {
-		qs[i] = "?"
-		in[i] = repo.FullName
-	}
-	return strings.Join(qs, ","), in
-}
+func toList(listof []*model.RepoLite) ([]string, [][]interface{}) {
+	var total = len(listof)
+	var limit = 999
+	var pagesCount = int(math.Ceil(float64(total) / float64(limit)))
 
-// helper function that converts a simple repository list
-// to a sql IN statement compatible with postgres.
-func toListPosgres(listof []*model.RepoLite) (string, []interface{}) {
```

Comment:
we should not combine `toList` and `toListPostgres` into a single function. If you look at the two functions, `toList` is more efficient because it doesn't need to grow the size of the string. The postgres version of the function needs to grow the size of the query string to account for variable digit size in `"$" + strconv.Itoa(i+1)`